server/index.js

```diff
@@ -1,4 +1,11 @@
 const express = require('express');
+const webpack = require('webpack')
+const webpackDevMiddleware = require('webpack-dev-middleware');
+const webpackHotMiddleware = require('webpack-hot-middleware');
+const webpackHotServerMiddleware = require('webpack-hot-server-middleware');
```

Comment:
Middleware to have hot reloading in the server, there are other ways to do this like using nodemon and having webpack running in watch mode emitting the files and let nodemon handle the server reload, but this more useful in my opinion.